javascript/advanced_examples/gdx_using_two_devices.html

```diff
@@ -161,45 +161,45 @@ <h1>Vernier Go Direct<sup>&reg</sup> JavaScript Advanced Example</h1>
 							labelString: 'Time'
 						}
 					}],
-					yAxes: [{
-						display: true,
-						scaleLabel: {
-							display: true,
-							labelString: 'Value'
-						}
-					}]
-				}
-			} 
-    };		
+        yAxes: [{
+          display: true,
+          scaleLabel: {
+            display: true,
+            labelString: 'Value'
+          }
+        }]
+      }
+    } 
+  };		
     
-    window.onload = function() {
-			var ctx = document.getElementById('canvas').getContext('2d');
-			window.myLine = new Chart(ctx, config);
-		}; 
-		
-		var colorNames = Object.keys(window.chartColors);
-	
-    selectDeviceBtn.addEventListener('click', selectCorrectDevice);
+  window.onload = function() {
+    var ctx = document.getElementById('canvas').getContext('2d');
+    window.myLine = new Chart(ctx, config);
```

Comment:
Same question about why this lives on the window 🤷‍♂️

Reply:
🤷‍♀